packages/synapse-react-client/src/style/components/_synapse-homepage.scss

```diff
@@ -9,6 +9,9 @@
     margin-top: 0;
     margin-bottom: 1rem;
   }
+  p.no-margin {
```

Comment:
I was unable to work around some of these scss styles with mui inline styles, so I needed to add a couple of classes